pom.xml

```diff
@@ -996,10 +996,9 @@ under the License.
 					<reuseForks>${flink.reuseForks}</reuseForks>
 					<systemPropertyVariables>
 						<forkNumber>0${surefire.forkNumber}</forkNumber>
-						<mvn.forkNumber>0${surefire.forkNumber}</mvn.forkNumber>
```

Comment:
Actually, this is probably fine. Could you re-add this line and remove the parameter from the the `argLine`?